consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -1952,6 +1961,11 @@ impl<
                         continue;
                     };
 
+                    // Skip if the epoch is not the current epoch
+                    if msg.epoch() != self.epoch {
+                        continue;
```

Comment:
we don't block in this file unlike the threshold_simplex voter. should we?

Reply:
https://github.com/commonwarexyz/monorepo/issues/1621

Reply:
We should (I thought I had an issue for this but regretfully did not)